netutils/config/parser.py

```diff
@@ -949,3 +949,77 @@ def _build_nested_config(self, line):
                 self.indent_level = spaces
 
             self._update_config_lines(line)
+
+
+class NokiaConfigParser(BaseSpaceConfigParser):
+    """Nokia SrOS config parser."""
+
+    comment_chars = ["# ", "#--------------------------------------------------"]
+    banner_start = []
+
+    def __init__(self, config):
+        """Create ConfigParser Object.
+
+        Args:
+            config (str): The config text to parse.
+        """
+        super(NokiaConfigParser, self).__init__(config)
+
+    def is_exit_next(self, line):  # pylint: disable=no-self-use
```

Comment:
I do not see this being used, I'm not sure how this is called?

Reply:
Sorry, this is left over from an earlier iteration. I had originally had it ignoring the `exit` clauses but have instead left them in. I've removed the method as it's no longer used.